plugins/filter/to_seconds.yml

```
@@ -0,0 +1,19 @@
+DOCUMENTATION:
+  name: community.general.to_seconds
+  short_description: Converte a date/time string to seconds
+  description:
+    - Parse a human readable date/time string and convert to seconds
```

There are no dates involved.
```suggestion
    - Parse a human readable time string and convert to seconds.
```